### PR TITLE
fix(Platform): Fix AOKZOE A1 support

### PR DESCRIPTION
**Pull request by pastaq**
* Fix incorrect platform for AOKZOE A1
* Fix orphaned_gamepad array being modified outside of mutex.lock()
* Fix bug where a device who's gamepad event was read before the keyboard device could fail to open properly.

### CODE REVIEW

File: core/global/input_manager.gd [hunk 1]
```diff
@@ -130,9 +130,9 @@ func _on_gamepad_change(_device: int, _connected: bool) -> void:
 			continue
 
 		logger.debug("Gamepad disconnected: " + gamepad.phys_path)
-		orphaned_gamepads[gamepad.phys] = gamepad
 		# Lock the gamepad mappings so we can alter them.
 		gamepad_mutex.lock()
+		orphaned_gamepads[gamepad.phys] = gamepad
```
Comment (ShadowApex) [inline core/global/input_manager.gd:135]: This is an unnecessary change as it doesn't appear that `orphaned_gamepads` is referenced by anything in a separate thread. Access to it appears to all be on the main thread.

**Review by ShadowApex — CHANGES_REQUESTED** [review]

File: core/global/input_manager.gd (hunk 1 above)
Comment (pastaq) [reply core/global/input_manager.gd:134]: reverted

**Review by ShadowApex — APPROVED** [review]
